src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -331,16 +331,6 @@ public OpenSearchSecurityPlugin(final Settings settings, final Path configPath)
             sm.checkPermission(new SpecialPermission());
         }
 
-        AccessController.doPrivileged(new PrivilegedAction<Object>() {
```

Comment:
@peternied @cwperks any reasons we need to explicitly add `BouncyCastleProvider`?

Reply:
Ahhh it is historical. It was added this way.

Reply:
From the javadoc for `java.security.Security` it is safe to call `addProvider` multiple times: https://docs.oracle.com/javase%2F7%2Fdocs%2Fapi%2F%2F/java/security/Security.html#addProvider(java.security.Provider)

> the preference position in which the provider was added, or -1 if the provider was not added because it is already installed.